### DISCUSSION TITLE
How to setup eslint LSP server

### DISCUSSION THREAD

**Discussion by batusai513** [LunarVim/LunarVim#2458] [Q&A] [answered]
I'm trying to setup the "native" [eslint server](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#eslint) from nvim-lsp-installer, I've installed the server, but it is not started when opening a js or ts file, when trying to update my config file adding this
```lua
 local opts = {
   format = false
 }
 require("lvim.lsp.manager").setup("eslint", opts)
```
eslint server is loaded but it says `Spawning language server with cmd: 'vscode-eslint-language-server' failed. The language server is either not installed, missing from PATH, or not executable.`.

Is there something I'm missing?

Thanks

**Comment by abzcoding** [comment]
we are using [nvim-lsp-intaller](https://github.com/williamboman/nvim-lsp-installer) as our language server management tool
so you'll need to install the language servers using that (hint: trying running `:LspInstallInfo` command and `?` key :) )

**Comment by kylo252** [comment, marked as answer]
This should be solved after https://github.com/LunarVim/LunarVim/pull/2243, specifically this commit: https://github.com/LunarVim/LunarVim/pull/2243/commits/5edbd910b3c2953e04da3caaee314ff8e861c053
